scripts/deployroot.js

```diff
@@ -0,0 +1,60 @@
+const hre = require('hardhat')
+require('dotenv').config()
+
+async function main () {
+  let fxRoot, checkpointManager
+
+  const network = await hre.ethers.provider.getNetwork()
+
+  if (network.chainId === 1) { // Ethereum Mainnet
+    fxRoot = '0xfe5e5D361b2ad62c541bAb87C45a0B9B018389a2'
+    checkpointManager = '0x86E4Dc95c7FBdBf52e33D563BbDB00823894C287'
+  } else if (network.chainId === 5) { // Goerli Testnet
+    fxRoot = '0x3d1d3E34f7fB6D26245E6640E1c50710eFFf15bA'
+    checkpointManager = '0x2890bA17EfE978480615e330ecB65333b880928e'
+  } else {
+    fxRoot = process.env.FX_ROOT
+    checkpointManager = process.env.CHECKPOINT_MANAGER
+  }
+
+  // You will want to use your own tunnel addresses here instead!
+  const ERC20 = await hre.ethers.getContractFactory('FxERC20RootTunnel')
+  const erc20 = await ERC20.deploy(checkpointManager, fxRoot, '0xAd87e3b217c66B0D45dEaFBC540330d300811b94')
```

Comment:
Can we make this configurable? from JSON file or something?

Reply:
Same for other addresses

Reply:
@jdkanani resolved!